index.html

```diff
@@ -41,5 +41,142 @@ <h1 class="header-text">Hey there. I’m Raphael
+            <h2>My Recent Works</h2>
+            <div class="bar"></div>
+            </div>
+            
+        </div>
+
+        <section class="cards-container">
+            <div class="card-item">
+                <div class="up-card"></div>
+                <div class="down-card">
+                    <h3>Multi-Post Stories Gain+Glory</h3>
+                    <ul class="skills"><li>Ruby on rails</li><li>CSS</li><li>JavScript</li><li>html</li></ul>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+
+            <div class="card-item">
+                
+                <div class="down-card">
+                    <h3>Multi-Post Stories Gain+Glory</h3>
+                    <ul class="skills"><li>Ruby on rails</li><li>CSS</li><li>JavScript</li><li>html</li></ul>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+
+            <div class="card-item">
+                
+                <div class="down-card">
+                    <h3>Multi-Post Stories Gain+Glory</h3>
+                    <ul class="skills"><li>Ruby on rails</li><li>CSS</li><li>JavScript</li><li>html</li></ul>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+            <div class="card-item">
+               
+                <div class="down-card">
+                    <h3>Multi-Post Stories Gain+Glory</h3>
+                    <ul class="skills"><li>Ruby on rails</li><li>CSS</li><li>JavScript</li><li>html</li></ul>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+
+            <div class="card-item">
+                
+                <div class="down-card">
+                    <h3>Multi-Post Stories Gain+Glory</h3>
+                    <ul class="skills"><li>Ruby on rails</li><li>CSS</li><li>JavScript</li><li>html</li></ul>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+            <div class="card-item">
+               
+                <div class="down-card">
+                    <h3>Multi-Post Stories Gain+Glory</h3>
+                    <ul class="skills"><li>Ruby on rails</li><li>CSS</li><li>JavScript</li><li>html</li></ul>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+        </section>
+
+
+        <section>
+            <div class="about-me">
+                <h2>About me</h2>
+                <div class="about-me-text">
+                 <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+                    </div>
+                    <div><button class="my-resume">Get My Resume</button></div>
```

Comment:
- [ ] [OPTIONAL] Good job adding different states to buttons as required by the previous reviewer. I think it will be very nice from a user experience point of view if you add `cursor: pointer;` to your buttons so the pointer can change when you are hovering over them. 😎️